### PR TITLE
Add db create

### PR DESCRIPTION
**Pull request by rob-duewer**
Ready for review

### CODE REVIEW

File: create-db-script/check-create-pgdb.sh [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+#CAF_DATABASE=
+#CAF_DATABASE_HOST=
+#CAF_DATABASE_PASSWORD=
+#CAF_DATABASE_PORT=
+#CAF_DATABASE_USERNAME=
+#ENV_PREFIX="CAF_"
+
+tmpDir="/tmp"
+scriptName=$(basename "$0")
+extension=$([[ "$scriptName" = *.* ]] && echo ".${scriptName##*.}" || echo '')
```
Comment (dermot-hardy) [inline create-db-script/check-create-pgdb.sh:13]: `extension` variable appears not to be used

File: create-db-script/check-create-pgdb.sh [hunk 2]
```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+# ----------Variable Section-----------#
```
Comment (dermot-hardy) [inline create-db-script/check-create-pgdb.sh:2]: Copyright header required - see [here](https://github.com/CAFapi/opensuse-tomcat-image/blob/v3.0.0/src/main/docker/startup.d/setup-tomcat-ssl-cert.sh) for example.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
The script itself seems fine but I think it should be put it straight into the [CAFapi/opensuse-tini-image](https://github.com/CAFapi/opensuse-tini-image) repository rather than having it here.  The certificate scripts are only in here for historical reasons and it's a pain because if they change there are more projects to release.  Create a `startup.d` directory in `opensuse-tini-image/src/main/docker/startup/` and put it in there.  The Dockerfile may also need to be adjusted (although looking at it I don't think it does).  And the release notes and readme should be updated for the new functionality.

**Comment by rob-duewer** [comment]
Cancelling pull request.  Will create a new one with changes against the opensuse-tini-image project instead.

**Pull request closed without merging**